.circleci/config.base.yml

```diff
@@ -305,6 +305,33 @@ jobs:
           path: ~/repo/packages/amplify-migration-tests/amplify-migration-reports
     working_directory: ~/repo
 
+  amplify_migration_tests_v10:
+    <<: *defaults
+    environment:
+      AMPLIFY_PATH: /home/circleci/.amplify/bin/amplify
+    steps:
+      - attach_workspace:
+          at: ./
+      - restore_cache:
+          key: amplify-category-api-yarn-deps-{{ .Branch }}-{{ checksum "yarn.lock" }}-{{ arch }}
+      - run:
+          name: Run tests migrating from CLI v10.5.1
+          command: |
+            yarn setup-dev
```

Comment:
I just realized that `yarn setup-dev` and setting `AMPLIFY_PATH=amplify-dev` (or so) should be default strategy everywhere.

We seem to be doing the following - we publish data packages to verdaccio and then install cli, which in theory should pull latest category api (as it uses `^`) .

However, when CLI ships with tagged release of category api (whether it's hot patch or CDK development) the version gets pinned (I.e. we don't `^`). In that case npm install -g of CLI won't pull latest api from verdaccio locally and e2e tests won't be testing changes...